### ISSUE TITLE
[iOS] Controls set to zero or near zero `Opacity` stop working

### ISSUE THREAD

**Issue by mikernet** [unoplatform/uno#7223]
### Current behavior

If you set a control like a button or a slider to `Opactiy=0` (or `0.001`, or `0.01`) they stop working. For example, we have an invisible slider overlaid over another control to allow finger sliding to set the display value of the control under it.

### Expected behavior

The controls should continue to work even though they are invisible. This works on UWP and WASM.

### How to reproduce it (as minimally and precisely as possible)

```xml
<Button Opacity="0" />
<Button Opacity="0.01" />
```

The button clicks stop working.

### Works on UWP/WinUI

Yes

### NuGet package version(s)

Uno.UI - v3.11.0-dev.210

### Affected platforms

iOS

**Comment by jeromelaban** [comment]
Thanks for the report.

Can you provide a repro sample for the behavior you are seeing ?

**Comment by mikernet** [comment]
Yup: https://github.com/mikernet/UnoTest/blob/40efc1c1874d79b09c1457409ca5ee0bed93443e/iRatio.Client/iRatio.Client.Shared/Roots/LoginRoot.xaml#L49

**Comment by mikernet** [comment]
Sorry give me a min, that iOS project has some issues.

**Comment by mikernet** [comment]
Do you need a working project or can you paste those few XAML lines and the handler into a test project on your end? There's something weird going on with the iOS project in that solution, it's not building right now.

**Comment by MartinZikmund** [comment]
Works correctly in Skia renderer targets
